index.html

```diff
@@ -0,0 +1,144 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+</head>
+<body>
+
+  <!-- HEADER SECTION -->
```

Comment:
remove all of those empty lines and comments

Reply:
pushed the changes